package.json

```diff
@@ -16,8 +16,7 @@
         "clean:node_modules": "pnpm dlx rimraf \"**/node_modules\" --glob",
         "lint": "biome check --diagnostic-level=error --max-diagnostics=200",
         "lint:fix": "biome check --write",
-        "publish": "lerna publish from-package",
-        "test": "lerna run test"
+        "test:vitest": "pnpm turbo test:vitest"
```

Comment:
why rename? to maintain consistency with nextjs app?

Reply:
yeah, and just "test" is pretty ambiguous